### PR TITLE
libchamplain: fix -devel depends

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

File: srcpkgs/libchamplain/template [hunk 1]
```diff
@@ -26,7 +26,7 @@ esac
 
 libchamplain-devel_package() {
 	depends="${sourcepkg}>=${version}_${revision} libglib-devel cairo-devel
-	 clutter-devel sqlite-devel libsoup-devel"
+	 clutter-gtk-devel sqlite-devel libsoup-devel"
```
Comment (maxice8) [inline srcpkgs/libchamplain/template]: the pkgconfnig file calls only for clutter-1.0

Comment (CameronNemo) [reply srcpkgs/libchamplain/template]: https://travis-ci.org/void-linux/void-packages/jobs/501311805

Comment (CameronNemo) [reply srcpkgs/libchamplain/template]: From `meson-log.txt`:

```
Dependency champlain-0.12 found: YES 0.12.19
Determining dependency 'champlain-gtk-0.12' with pkg-config executable '/usr/bin/pkg-config'
Called `/usr/bin/pkg-config --modversion champlain-gtk-0.12` -> 0
0.12.19
Called `/usr/bin/pkg-config --cflags champlain-gtk-0.12` -> 1
```

**Comment by maxice8** [comment]
On Sun, Mar 03, 2019 at 08:39:40PM -0800, Cameron Nemo wrote:
> CameronNemo commented on this pull request.
> 
> 
> 
> > @@ -26,7 +26,7 @@ esac
>  
>  libchamplain-devel_package() {
>  	depends="${sourcepkg}>=${version}_${revision} libglib-devel cairo-devel
> -	 clutter-devel sqlite-devel libsoup-devel"
> +	 clutter-gtk-devel sqlite-devel libsoup-devel"
> 
> https://travis-ci.org/void-linux/void-packages/jobs/501311805
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/9347#discussion_r261920959
 
Please add it beside clutter-devel. I'll file a bug report upstream.

File: srcpkgs/libchamplain/template (hunk 1 above)
Comment (maxice8) [reply srcpkgs/libchamplain/template]: Please also add gtk+3-devel

**Comment by CameronNemo** [comment]
No bug upstream, just another pc file.

```
$ cat /usr/lib/pkgconfig/champlain-gtk-0.12.pc 
prefix=/usr
libdir=${prefix}/lib
includedir=${prefix}/include

Name: champlain-gtk-0.12
Description: Gtk+ Widget wrapper for libchamplain
Version: 0.12.19
Requires: gtk+-3.0 >=  3.0, clutter-gtk-1.0 >=  1.0, champlain-0.12
Requires.private: glib-2.0 >=  2.38, gobject-2.0 >=  2.38, cairo >=  1.4, clutter-1.0 >=  1.24
Libs: -L${libdir} -lchamplain-gtk-0.12
Cflags: -I${includedir}/champlain-0.12
```

File: srcpkgs/libchamplain/template (hunk 1 above)
Comment (maxice8) [reply srcpkgs/libchamplain/template]:
```
$ xpcdeps champlain-gtk-0.12
cairo-devel -> cairo
clutter-devel -> clutter-1.0
clutter-gtk-devel -> clutter-gtk-1.0
gtk+3-devel -> gtk+-3.0
libchamplain-devel -> champlain-0.12
libglib-devel -> glib-2.0
libglib-devel -> gobject-2.0
```

Comment (CameronNemo) [reply srcpkgs/libchamplain/template]: done

**Pull request merged**